e2e-tests/auth/selfsigned/apps/selfsigned.go

```diff
@@ -91,12 +92,28 @@ func (s SelfSigned) GetSessionStatus(sessionID string) (string, *authAPI.Verifia
 	return response.JSON200.Status, response.JSON200.VerifiablePresentation, nil
 }
 
-func (s SelfSigned) RequestAccessToken(organizationDID string, purposeOfUse string, presentation *authAPI.VerifiablePresentation) (*authAPI.TokenIntrospectionResponse, error) {
+func (s SelfSigned) UntrustEmployeeCredential(issuerDID string) error {
```

Comment:
only used in tests?

Reply:
hmm.. having e2e-tests in the main repo does require you to look twice sometimes